metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -387,18 +388,32 @@
 bolt_hdfs_rotation_policy_units = config['configurations']['metron-indexing-env']['bolt_hdfs_rotation_policy_units']
 bolt_hdfs_rotation_policy_count = config['configurations']['metron-indexing-env']['bolt_hdfs_rotation_policy_count']
 
-# Pcap
+# PCAP
 metron_pcap_topology = 'pcap'
 pcap_input_topic = 'pcap'
```

Comment:
pcap input topic is also exposed , so shouldn't we be getting the value from config tab ?

Reply:
pcap input topic is also hardcoded in [status_param.py ](https://github.com/MohanDV/metron/blob/a46fda45b46a9f317e2ef7c3755ce6f072ed893b/metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/status_params.py#L123)  we should get the value from config tab here too.